Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+ENV KRSV2_DBURL '< DB URL HERE >'
+
+COPY . .
+
+EXPOSE 6833
+
+CMD [ "npm", "start" ]
```

Comment:
Can you add to the project's README that describe how a developer can pull down this project and spin it up locally using docker?

Reply:
Added here:
https://github.com/BitGo/key-recovery-service-v2/pull/31/commits/f2f909823d00674277107add97fbbbb313d90867